modules/apps/login/login-web/src/main/resources/META-INF/resources/login.jsp

```diff
@@ -219,6 +219,13 @@
 					});
 				}
 			}
+			AUI().ready(function (A) {
+				document.getElementsByClassName('btn disabled btn-primary')[0].disabled =
+					false;
+				document
+					.getElementsByClassName('btn disabled btn-primary')[0]
+					.classList.remove('disabled');
```

Comment:
probably is better to save the element in a variable so we don't need to parse the DOM twice for the same thing